packages/terra-functional-testing/package.json

```diff
@@ -89,13 +89,14 @@
     "webpack-cli": "^3.3.12 || ^4.0.0"
   },
   "devDependencies": {
+    "@babel/cli": "^7.20.7",
     "@cerner/terra-cli": "^1.9.0",
     "chai": "^4.2.0",
     "lodash.groupby": "^4.6.0",
     "lodash.map": "^4.6.0",
     "lodash.mapkeys": "^4.6.0",
     "lodash.mapvalues": "^4.3.2",
-    "mocha": "^8.1.1",
+    "mocha": "^10.2.0",
```

Comment:
have we gone through the upgrade guide or change log of mocha to ensure no code changes are required.